packages/f-content-cards/src/components/cardTemplates/VoucherCard.vue

```diff
@@ -43,6 +44,12 @@ export default {
         };
     },
 
+    computed: {
+        isAnniversaryCard () {
+            return this.cardType === 'Anniversary_Card_1';
```

Comment:
I think you need to account for the transformation done by normaliseCardType here:
```suggestion
            return this.cardType === 'anniversaryCard1';
```